test/test_nnapi.py

```diff
@@ -503,6 +503,62 @@ def test_conv2d(self):
                         limit=limit
                     )
 
+    def test_conv2d_transpose(self):
+        weight = torch.randn((128, 256, 2, 2))
+        input_dim = (30, 128, 7, 7)
```

Comment:
Do we need the data to be this large?

Reply:
Definitely not, was dealing with that size during conversion ended up using it in the test too